README.md

```diff
@@ -84,6 +84,9 @@
 - [https://github.com/Ghazif-Adeem/COVID-19-Statistics-Checker] (https://github.com/Ghazif-Adeem/COVID-19-Statistics-Checker), (PHP) by [Ghazif Adeem](https://github.com/Ghazif-Adeem)
 
 
+- [https://krishnatre-siddhartha.github.io/] (https://krishnatre-siddhartha.github.io/), (HTML+JS) by [Siddhartha Sharma](https://github.com/krishnatre-siddhartha/)
```

Comment:
I think hyperlink markdown shouldn't have a space between a label and link.
![image](https://user-images.githubusercontent.com/5756522/79132957-7143db80-7dd5-11ea-843b-554a4b264201.png)